### ISSUE TITLE
/!\ ffmpeg fail, job commandline: M1 MacOS

### ISSUE THREAD

**Issue by Smiril** [chychkan/DeepFaceLab_MacOS#102]
well failed without error code 

have try following:

psutil>=5.8.0
numpy==1.22.0
numexpr<=2.8.1
opencv-python<=4.6.0.66
scikit-image<=0.19.3
scikit-learn<=1.0.2
scipy<=1.8.1
tifffile<=2022.5.4
pyparsing==3.0.9
pillow
tqdm<=4.64.0
colorama<=0.4.4
cython<=0.29.26
PyWavelets<=1.3.0
imageio<=2.19.3
networkx<=2.8.4
packaging<=21.3
image
timm
PlL
moviepy
absl-py<=1.1.0
astunparse
cachetools<=5.2.0
certifi<=2022.6.15
charset-normalizer<=2.1.0
flatbuffers<=1.12
gast
google-auth
google-auth-oauthlib<=0.4.6
google-pasta<=0.2.0
grpcio
h5py<=3.7.0
idna<=3.3
keras<=2.9.0
keras-preprocessing
libclang<=14.0.1
markdown<=3.3.7
oauthlib<=3.2.0
opt-einsum<=3.3.0
pyasn1==0.4.8
pyasn1-modules<=0.2.8
requests<=2.28.1
requests-oauthlib<=1.3.1
rsa<=4.8
six
Flask<=2.1.3
Jinja2<=3.1.2
Mako<=1.2.1
MarkupSafe<=2.1.1
alembic<=1.8.1
click<=8.1.3
cloudpickle<=2.1.0
databricks-cli
docker<=5.0.3
entrypoints<=0.4
gitdb<=4.0.9
gitpython<=3.1.27
gunicorn<=20.1.0
importlib-metadata<=4.12.0
itsdangerous<=2.1.2
mlflow<=1.27.0
pandas<=1.4.3
prometheus-client<=0.14.1
prometheus-flask-exporter<=0.20.2
pyjwt<=2.4.0
python-dateutil<=2.8.2
pytz<=2022.1
pyyaml<=6.0
querystring-parser<=1.2.4
smmap<=5.0.0
sqlalchemy<=1.4.39
sqlparse<=0.4.2
tabulate<=0.8.10
websocket-client<=1.3.3
zipp<=3.8.1
tensorboard<=2.9.1
tensorboard-data-server<=0.6.1
tensorboard-plugin-wit<=1.8.1
tensorflow-estimator<=2.9.0
tensorflow-macos<=2.9.0 
tensorflow-metal
termcolor<=1.1.0
typing-extensions<=4.3.0
urllib3<=1.26.10
werkzeug<=2.1.2
wrapt<=1.14.1
fastcluster>=1.2.4
matplotlib>=3.5.1
ffmpeg-python==0.2.0
imageio-ffmpeg>=0.4.7
ffmpy
torch<=1.12.0
torchvision<=0.13.0
torchaudio<=0.12.0
insightface<=0.2.1
ipython<=7.21.0
PyQt6<=6.2.3
git+https://www.github.com/keras-team/keras-contrib.git

` python3 /Users/Smiril/Documents/deep/DeepFaceLab_MacOS/.dfl/DeepFaceLab/main.py videoed extract-video --input-file /Users/Smiril/Documents/deep/DeepFaceLab_MacOS/workspace/data_src.mp4 --output-dir /Users/Smiril/Documents/deep/DeepFaceLab_MacOS/workspace/data_src/ --output-ext png --fps 60
Metal device set to: Apple M1 Max

systemMemory: 64.00 GB
maxCacheSize: 24.00 GB

/!\ ffmpeg fail, job commandline:['ffmpeg', '-i', '/Users/Smiril/Documents/deep/DeepFaceLab_MacOS/workspace/data_src.mp4', '-pix_fmt', 'rgb24', '-r', '60', '/Users/Smiril/Documents/deep/DeepFaceLab_MacOS/workspace/data_src/%5d.png']
Done.`

could anyone help?

**Comment by jeremybarbaud** [comment]
At time the fix is : 
deactivate 
source .dfl/env/bin/activate

2x time
